packages/h2-lwt-unix/h2-lwt-unix.0.5.0/opam

```diff
@@ -0,0 +1,33 @@
+opam-version: "2.0"
+maintainer: "Antonio Monteiro <anmonteiro@gmail.com>"
+authors: [ "Antonio Monteiro <anmonteiro@gmail.com>" ]
+license: "BSD-3-clause"
+homepage: "https://github.com/anmonteiro/ocaml-h2"
+bug-reports: "https://github.com/anmonteiro/ocaml-h2/issues"
+dev-repo: "git+https://github.com/anmonteiro/ocaml-h2.git"
+doc: "https://anmonteiro.github.io/ocaml-h2/"
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+]
+depends: [
+  "ocaml" {>= "4.06"}
+  "faraday-lwt-unix"
+  "h2-lwt" {= version}
+  "dune" {>= "1.5"}
```

Comment:
```suggestion
  "dune" {>= "1.7"}
```
Dune 1.5 won't work as the `fmt.1.1` plugin has only been there since dune 1.7 and dune does not warn users about this (see https://github.com/ocaml/dune/issues/2957)